index.js

```diff
@@ -1,8 +1,5 @@
 'use strict';
 
-const fromEntries = require('object.fromentries');
-const entries = require('object.entries');
```

Comment:
this plugin supports node 4, which lacks Object.entries, so we still need the polyfill.